test/integration/knative_test.go

```diff
@@ -33,6 +33,10 @@ const (
 )
 
 func TestKnativeIngress(t *testing.T) {
+	if clusterVersion.Major < uint64(1) || (clusterVersion.Major == uint64(1) && clusterVersion.Minor < uint64(21)) {
```

Comment:
Does https://pkg.go.dev/github.com/blang/semver#Version.LT not work here for some reason?

Reply:
No that's fine I just forgot about it, changed in f54e25f9079d91e288c13e420b6f3842074b255b